source/manage/telemetry.rst

```diff
@@ -138,7 +138,7 @@ Non-personally Identifiable Diagnostic Information, distinguished by end users a
   - *Commercial License Diagnostics (Enterprise Edition Only):* Uploaded an Enterprise license key to the server
   - *Mobile Performance Diagnostics:* Load times for starting the app, switching channels, and switching teams
   - *Permissions Discovery Diagnostics (Enterprise Edition Only):* Provides all the permissions configured for each role for the System Scheme and each Team Override Scheme created in the system. Scheme ID; Team Admin permissions; Team user permissions; Channel Admin permissions; Channel user permissions; Number of teams the scheme is associated with
-  - *Group Discovery Diagnostics (Enterprise Edition Only):* Provides information related to AD/LDAP and custom groups, including number of groups synced to Mattermost, teams and channels associated to groups, teams and channels synced with groups, and number of group members
+  - *Group Discovery Diagnostics (Enterprise Edition Only):* Provides information related to AD/LDAP and custom groups, including number of unique users in groups, number of groups synchronized to Mattermost, teams and channels associated to groups, teams and channels synchronized with groups, and number of group members.
```

Comment:
Just a reminder that Custom Groups applies to `Professional` and `Enterprise` licenses.

Reply:
Great reminder, @mkraft! Thank you! I'll get this updated.